src/openrct2-ui/interface/InGameConsole.cpp

```diff
@@ -176,6 +177,12 @@ void InGameConsole::ClearLine()
 
 void InGameConsole::Open()
 {
+    if (!_isInitialised)
+    {
+        WriteInitial();
+        _isInitialised = true;
+    }
```

Comment:
I wonder if it would be better to keep all of this logic on `WriteInitial()`

Reply:
Not a fan. I will change it if you insist, though.

Reply:
Can't we just make the initialization of our subsystems explicit? If console depends on something then we need that to be initialized later.

Reply:
I think I see where the issue is at, https://github.com/OpenRCT2/OpenRCT2/blob/dec5140f20d57cf3a534ff4ee7685c1f78298aee/src/openrct2-ui/interface/InGameConsole.cpp#L27
There is an unused static here, ordinarily it will be initialized via the Context in where it should be fine, can you try deleting that line and see if that helps.

Reply:
Additionally https://github.com/OpenRCT2/OpenRCT2/blob/dec5140f20d57cf3a534ff4ee7685c1f78298aee/src/openrct2/Context.cpp#L161 this should be the second thing to be initialized. A lot of things rely on localisation so it should be available as early as possible.

Reply:
I tried applying both changes, but that didn’t help. Do you have another idea?

Reply:
I looked into it, apparently not everything is created within the Context, this is rather messy. I guess this will have to do for now.

Reply:
Very well, could I have an approval then, @tupaschoal , or would you like me to make the change you mentioned?